samigo/samigo-app/src/webapp/jsf/author/item/tags.jsp

```diff
@@ -1,5 +1,5 @@
 <script type="text/javascript">includeWebjarLibrary('select2');</script>
-<script type="text/javascript" src="/library/webjars/select2/4.0.8/js/i18n/<h:outputText value='#{itemauthor.language}' rendered='#{not empty itemauthor.language}' /><h:outputText value='en' rendered='#{empty itemauthor.language}' />.js"></script>
+<script type="text/javascript" src="/library/webjars/select2/4.0.12/js/i18n/<h:outputText value='#{itemauthor.language}' rendered='#{not empty itemauthor.language}' /><h:outputText value='en' rendered='#{empty itemauthor.language}' />.js"></script>
```

Comment:
Haha, you don't need the type attribute. I knew you'd forget :)

Reply:
You're right!